Formula/rsyslog.rb

```diff
@@ -1,8 +1,8 @@
 class Rsyslog < Formula
   desc "Enhanced, multi-threaded syslogd"
   homepage "https://www.rsyslog.com/"
-  url "https://www.rsyslog.com/files/download/rsyslog/rsyslog-7.4.5.tar.gz"
-  sha256 "f5e46e9324e366f20368162b4f561cf7a76fecb4aa0570edcaaa49e9f8c2fe70"
+  url "http://www.rsyslog.com/files/download/rsyslog/rsyslog-8.34.0.tar.gz"
```

Comment:
What's wrong with the https link?

Reply:
Oversight on my part, fixed it in latest push.